src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(item => (
+      <MovieCard {...item} key={item.imdbId} />
```

Comment:
When spreading props like this, you pass all the object's properties to the component. This can lead to prop pollution and make it harder to reason about the component's dependencies.
![image](https://github.com/mate-academy/react_movies-list/assets/99403198/078820a7-a6cf-4270-90ad-74c3f5810457)

```suggestion
    {movies.map(movie => (
      <MovieCard movie={movie} key={item.imdbId} />
```